contracts/axelarnet-gateway/src/contract/execute.rs

```diff
@@ -115,10 +119,18 @@ pub fn call_contract(
         token: token.clone(),
     };
 
-    let res = match determine_routing_destination(&client, &msg.destination_chain)? {
-        RoutingDestination::Nexus => route_to_nexus(&client, nexus, msg, token)?,
+    let res = match determine_routing_destination(
```

Comment:
Just call `route_messages` here instead. Fine for it to check if it's a routable_msg again like it was already doing